.github/CODEOWNERS

```diff
@@ -8,6 +8,9 @@
 # Catch all
 /sdk/                                               @joshfree @jonathangiles
 
+# BOM
+/sdk/boms/azure-sdk-bom/                            @alzimmermsft @jonathangiles
```

Comment:
Should this be `boms` or `bom`?

Reply:
@mitchdenny asked for this to be placed under /boms. Mitch, does this align with directory changes to support Spring BOMs?

Reply:
Well the parent POM restructuring change I have (when I eventually land it) will put parent poms in sdk/parents. So in sticking with pluralization sdk/boms :)